generator/generator.yml

```diff
@@ -400,3 +400,43 @@ modules:
         type: EnumAsStateSet
     auth:
       community: raritan_public
+
+# InfraPower PDU's
+#
+# https://www.austin-hughes.com/support/software/infrapower/IPD-MIB.7z
+  infrapower_pdu:
+    walk:
+      - sysUpTime
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.1 # pduIndex
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.6 # pduName
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.7 # pduLocation
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.13 # pduMainLoadVoltage
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.14 # pduMainLoadAmp
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.17 # pduMainActivePower
+    lookups:
+      - source_indexes: [pduIndex]
+        lookup: pduName
+      - source_indexes: [pduIndex]
+        lookup: pduLocation
+    overrides:
+      pduIndex:
+        ignore: true # Lookup metric
+      pduName:
+        ignore: true # Lookup metric
+      pduLocation:
+        ignore: true # Lookup metric
+      pduMainLoadAmp:
+        regex_extracts:
+          Fix:
```

Comment:
You can leave this as the empty string

Reply:
So just remove Fix?

Reply:
Yes. It is this metric, there's no "Fix" information that has been extracted from it.